builtin/credential/marathon/backend_test.go

```diff
@@ -0,0 +1,64 @@
+package marathon
```

Comment:
added a test for this. just need to run it like this with `big up -d marathon` running:

```bash
$ MARATHON_HOST=marathon.dev.banno.com make testacc TEST=./builtin/credential/marathon
```

The test can be flaky because im pulling down `small-deployable` docker container and running that. Probably not ideal...